sdk/python/scripts/ci/test_vectors.sh

```diff
@@ -2,7 +2,9 @@
 
 set -ex
 
-PYTHONPATH=. SCHEMAS_PATH="$(git rev-parse --show-toplevel)/tests/vectors" python3 -m pytest tests/generator
+TEST_RUNNER=$([ "$1" = "code-coverage" ] && echo "coverage run --append" || echo "python3")
```

Comment:
is it intentional that this is only command with `--append` ?

Reply:
Yes, by default code coverage tool will overwrite the previous run.  
The other option which I was also thinking would be to generate a different code coverage file for each run and merged them  at the end.

Reply:
shouldn't js test-vectors also have append or does tool automatically merge?

Reply:
its part of the script, there is a ``--no-clean`` flag.  
I am think I should probably add the ``append`` flag to all of the code coverage run so order would matter.  The first run will should always be in a clean workspace so it shouldnt matter.  Your thoughts?